### PR TITLE
Increase runtime code coverage after removing CLI tool

### PR DESCRIPTION
**Pull request by dilanSachi**
## Purpose
$subject
Fixes https://github.com/ballerina-platform/ballerina-standard-library/issues/3890
## Checklist
- [x] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: native/src/test/java/io/ballerina/stdlib/grpc/MessageRegistryTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package io.ballerina.stdlib.grpc;
```
Comment (MadhukaHarith92) [inline native/src/test/java/io/ballerina/stdlib/grpc/MessageRegistryTest.java:1]: License header missing

File: native/src/test/java/io/ballerina/stdlib/grpc/ServerCallTest.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
+package io.ballerina.stdlib.grpc;
```
Comment (MadhukaHarith92) [inline native/src/test/java/io/ballerina/stdlib/grpc/ServerCallTest.java:1]: License header missing

File: native/src/test/java/io/ballerina/stdlib/grpc/nativeimpl/serviceendpoint/FunctionUtilsTest.java [hunk 3]
```diff
@@ -0,0 +1,214 @@
+package io.ballerina.stdlib.grpc.nativeimpl.serviceendpoint;
```
Comment (MadhukaHarith92) [inline native/src/test/java/io/ballerina/stdlib/grpc/nativeimpl/serviceendpoint/FunctionUtilsTest.java:1]: License header missing

File: native/src/test/java/io/ballerina/stdlib/grpc/nativeimpl/streamingclient/FunctionUtilsTest.java [hunk 4]
```diff
@@ -0,0 +1,45 @@
+package io.ballerina.stdlib.grpc.nativeimpl.streamingclient;
```
Comment (MadhukaHarith92) [inline native/src/test/java/io/ballerina/stdlib/grpc/nativeimpl/streamingclient/FunctionUtilsTest.java:1]: License header missing

File: native/src/test/java/io/ballerina/stdlib/grpc/MessageRegistryTest.java (hunk 1 above)
Comment (dilanSachi) [reply native/src/test/java/io/ballerina/stdlib/grpc/MessageRegistryTest.java:19]: Added

File: native/src/test/java/io/ballerina/stdlib/grpc/ServerCallTest.java (hunk 2 above)
Comment (dilanSachi) [reply native/src/test/java/io/ballerina/stdlib/grpc/ServerCallTest.java:19]: Added

File: native/src/test/java/io/ballerina/stdlib/grpc/nativeimpl/caller/FunctionUtilsTest.java [hunk 5]
```diff
@@ -0,0 +1,59 @@
+package io.ballerina.stdlib.grpc.nativeimpl.caller;
```
[the remark this answers is not in the document]
Comment (dilanSachi) [reply native/src/test/java/io/ballerina/stdlib/grpc/nativeimpl/caller/FunctionUtilsTest.java:19]: Added

File: native/src/test/java/io/ballerina/stdlib/grpc/nativeimpl/streamingclient/FunctionUtilsTest.java (hunk 4 above)
Comment (dilanSachi) [reply native/src/test/java/io/ballerina/stdlib/grpc/nativeimpl/streamingclient/FunctionUtilsTest.java:19]: Added

File: native/src/test/java/io/ballerina/stdlib/grpc/nativeimpl/serviceendpoint/FunctionUtilsTest.java (hunk 3 above)
Comment (dilanSachi) [reply native/src/test/java/io/ballerina/stdlib/grpc/nativeimpl/serviceendpoint/FunctionUtilsTest.java:19]: Added

**Review by MadhukaHarith92 — APPROVED** [review]
LGTM

**Pull request merged**